packages/match-media/src/index.ts

```diff
@@ -25,9 +25,13 @@ export const useBreakpointIndex = (options: defaultOptions = {}) => {
   const [value, setValue] = useState(defaultIndex)
   useEffect(() => {
     const getIndex = () =>
-      breakpoints.filter(
-        (bp) => window.matchMedia(`screen and (min-width: ${bp})`).matches
-      ).length
+      breakpoints.filter((bp) => {
+        const query = bp.includes('@media')
+          ? bp.slice(7) // remove '@media '
```

Comment:
`bp.replace('@media ', '')` might be more readable, I don't feel strongly though

Reply:
Fair point! My code is actually buggy! If you have a leading space (like `  @media `), my version wouldn't work. Applying your suggestion!